tests/Api/Admin/CatalogPromotionsTest.php

```diff
@@ -160,6 +160,67 @@ public function it_does_not_create_a_catalog_promotion_with_taken_code(): void
         );
     }
 
+    /** @test */
+    public function it_does_not_create_a_catalog_promotion_with_invalid_actions(): void
+    {
+        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'channel.yaml', 'product_variant.yaml']);
+        $header = $this->getLoggedHeader();
+
+        $this->client->request(
+            'POST',
+            '/api/v2/admin/catalog-promotions',
+            [],
+            [],
+            $header,
+            json_encode([
+                'name' => 'T-Shirts discount',
```

Comment:
I think we should consider something for our contract tests that will allow us to use json files with request body. Because those tests are getting enormously huge with just this funky 💃 code.

Reply:
We can probably use json files without problems, but then these tests will become less readable, so I'm not 100% convinced. We should think about that and probably refactor our contracts tests in a separate PR